comp/serializer/compression/compressionimpl/common.go

```diff
@@ -0,0 +1,16 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package compressionimpl provides a set of functions for compressing with zlib / zstd
+package compressionimpl
+
+// ZlibKind defines a const value for the zlib compressor
+const ZlibKind = "zlib"
+
+// ZstdKind  defines a const value for the zstd compressor
+const ZstdKind = "zstd"
+
+// NoneKind defines a const value for disabling compression
+const NoneKind = "none"
```

Comment:
Today these are private to the implementation, and by design the interface doesn't care about what the underlying implementation ends up being (from build tags, or config). Perhaps this could change in the future - but this is why I put them in the impl today. 

With this in mind, do you still think I should move them to the interface?

Reply:
Thanks for the context. In that case it doesn't make sense to move to the component interface package